apps/condo/domains/resident/gql.js

```diff
@@ -10,7 +10,7 @@ const gql = require('graphql-tag')
 
 const COMMON_FIELDS = 'id dv sender v deletedAt newId createdBy { id name } updatedBy { id name } createdAt updatedAt'
 
-const RESIDENT_FIELDS = `{ user { id name } organization { id } property { id } billingAccount { id } address addressMeta unitName ${COMMON_FIELDS} }`
+const RESIDENT_FIELDS = `{ user { id name } organization { id } residentOrganization { id name } property { id } residentProperty { id name } billingAccount { id } address addressMeta unitName ${COMMON_FIELDS} }`
```

Comment:
add property resident address

Reply:
We already have `address` property, it's a scalar.

Reply:
Added in `a6160203`